apps/src/blockly/addons/functionEditor.js

```diff
@@ -159,25 +159,28 @@ export default class FunctionEditor {
       Blockly.getHiddenDefinitionWorkspace()
     );
 
+    let type;
```

Comment:
is there a bug fix with the changes around `type` here or is it just a readability fix?

Reply:
No bug here.

These two lines were further down
```
const type = procedureType || existingProcedureBlock.type;
    const isBehavior = type === BLOCK_TYPES.behaviorDefinition;
```

And I had to look up this code to double check how `procedureType` and `existingProcedureBlock.type` are used, and if we moved `isBehavior` up, we could use it in the delete button logic, where we were also doing a conditional on block type.